src/components/StopWatch.tsx

```diff
@@ -1,26 +1,33 @@
 import { useEffect, useState } from "react";
+import { Button } from "./Button";
 import { Timer } from "./Timer";
 export const StopWatch = () => {
```

Comment:
`StopWatch: React.FC` it's better to add this type to make sure other contributors immediately know this is a React functional component